cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -108,8 +110,8 @@ data:
       @type detect_exceptions
 
       remove_tag_prefix raw
-      message log
```

Comment:
This line is different that the one in the original PR. Is this intended? If yes please explain why in the PR description.

Reply:
This is only different because it has been merged in master between 1.10 and 1.11, this is the intended value to properly capture exceptions.